web/app/js/index.js

```diff
@@ -36,18 +36,23 @@ import { dashboardTheme } from './components/util/theme.js';
 const appMain = document.getElementById('main');
 const appData = !appMain ? {} : appMain.dataset;
 
-let pathPrefix = '';
+let pathPrefix = appData.pathPrefix;
 const proxyPathMatch = window.location.pathname.match(/\/api\/v1\/namespaces\/.*\/proxy/g);
 if (proxyPathMatch) {
   pathPrefix = proxyPathMatch[0];
 }
 
+// Make sure the pathPrefix does not end with a '/'
+if (pathPrefix.slice(-1) === '/') {
+  pathPrefix = pathPrefix.slice(0, -1);
+}
+
 let defaultNamespace = 'default';
-const pathArray = window.location.pathname.split('/');
+const pathArray = window.location.pathname.substring(pathPrefix.length).split('/');
 
 // if the current URL path specifies a namespace, this should become the
 // defaultNamespace
-if (pathArray[0] === '' && pathArray[1] === 'namespaces' && pathArray[2]) {
+if (pathPrefix !== '' && pathArray[0] === '' && pathArray[1] === 'namespaces' && pathArray[2]) {
```

Comment:
This doesn't seem quite right – you're making the entire check for a namespace in the path conditional on the path prefix being set?